tests/acceptance/course-admin/view-code-example-test.js

```diff
@@ -0,0 +1,41 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
+import { signInAsStaff } from 'codecrafters-frontend/tests/support/authentication-helpers';
+import catalogPage from 'codecrafters-frontend/tests/pages/catalog-page';
+import courseOverviewPage from 'codecrafters-frontend/tests/pages/course-overview-page';
+import coursePage from 'codecrafters-frontend/tests/pages/course-page';
+import createCommunityCourseStageSolution from 'codecrafters-frontend/mirage/utils/create-community-course-stage-solution';
+import testScenario from 'codecrafters-frontend/mirage/scenarios/test';
+
+module('Acceptance | course-admin | view-code-example', function (hooks) {
```

Comment:
This isn't testing the admin area, is it? I thought there's an equivalent test for this inside the current "view-code-examples" test - @VasylMarchuk can you check?

Reply:
As agreed, will do this in a follow-up PR, already preparing it right now.